pkg/source/markdown.go

```diff
@@ -202,3 +221,33 @@ func (m *markdown) parse(filename string, opts Opts) (err error) {
 
 	return
 }
+
+func (m *markdown) parseDirectives() error {
+	firstDirective, optionalDirectives := m.directives[0], m.directives[1:]
```

Comment:
This approach does not work when dox is run on a "new" repo. For example, a dox header of `dox: omit-notice` is valid the first time, but this does not account for that.